src/components/ExpandedControlsPanel.module.css

```
@@ -0,0 +1,117 @@
+
+/* Key column base styles */
+:global([data-layout-column="key"]) {
+  position: relative;
+  align-items: center;
+}
+
+:global([data-layout-column="key"]) > h2 {
+  display: block;
+  text-align: center;
+}
+/* stylelint-enable selector-pseudo-class-no-unknown */
```

Stylelint (17.7.0)

[error] "selector-pseudo-class-no-unknown" has not been disabled (CssSyntaxError)

(CssSyntaxError)